Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
Oh, it sounds like `CHANGE_AUTHOR` is one piece of information that the PR builder gives... Maybe it also gets the branch name?